### ISSUE TITLE
Staging Review finding: Instructions after the inputs may be missed by screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#103788]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Dependent Experience Team
**Product:** 21-686c and 21-674
**Feature:** Critical Form Updates

## Findings details
**VA.gov Experience Standard - issue:** [User has enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/current-marriage-information/type-of-marriage](https://staging.va.gov/view-change-dependents/add-remove-form-21-686c-v2/current-marriage-information/type-of-marriage) for the question "How did you get married?" each response generates some chunk of further instructions --- ranging from a short "we'll need documents at the end" to lengthy descriptions of the documents that must be provided. This text is added conditionally when you make your selection.

Like anyone else moving through a form flow, we expect screen reader users to be moving through it as efficiently as possible --- pressing [tab] to move through the focusable elements on the page, rather than having their screen reader announce the full content of the page after each selection. That means that without the visual cue of something being added to the page screen reader users are likely to miss these instructions entirely. They'll tab straight to the next element and never know that text was added.

Since you'll be prompted to provide the documents later, I don't think this is a launch blocker. But it is a noticeably degraded experience for screen reader users.

Similar issues occur elsewhere when instructions are placed after the inputs:
- [/686-report-add-child/1/place-of-birth](https://staging.va.gov/view-change-dependents/add-remove-form-21-686c-v2/686-report-add-child/1/place-of-birth?add=true) for "Based on your answers, you'll need to submit..."

<details><summary>Link, screenshot or steps to recreate</summary>

!["How did you get married?" question with "In a tribal ceremony" selected. The question is followed by several paragraphs of new instructions.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/74059-Brian-1.png?raw=true)

</details>

## Recommended action

Two possible recommendations here:
- Move the revealed instructions to a new page that appears when you click Continue, rather than having the instructions appear beneath the question. If a screen reader comes to a page in the form flow that doesn't have any inputs they can tab to, that's more likely to prompt them to explore the content of the page and discover these instructions.
- If it makes more sense to keep these instructions on the same page, wrap them in a `div aria-live="polite"`. An [`aria-live` region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live) signals to screen readers to announce whenever something changes in that region, so any new text added there will be announced without needing to do any exploration. (Note that when no input is selected you'll still need an empty div, `<div aria-live="polite"></div>`, to set a baseline state.)

Either of these are fine, no strong preference for which you choose.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by Midge-dev** [comment]
Opt for wrapping in aria-live=polite
